LICENSES/README.md

```diff
@@ -1,8 +1,8 @@
-# Hirte Project Licensing
+# Bluechi Project Licensing
```

Comment:
C is capatalized I believe?

Reply:
@rhatdan Sure, I will take care of all, regarding docs and md files. Thanks!